spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module FeatureHelpers
+  def sign_in_with_user(user_email_address)
+    mock_auth_with_user(user_email_address)
+    visit sign_in_path
+    click_button("Sign in with your DfE Microsoft account")
```

Comment:
It would be worth using the locale here, incase we change it, one less thing to do.

`click_button(I18n.t("sign_in.button"))`